test_utils/basetest.py

```diff
@@ -244,10 +244,47 @@ def fetchlogs(self, app_insights):
         return result
 
     def filter_logs(self, logs, key, value):
-        return value in [d.get(key) for d in logs]
+        return [d.get(key) for d in logs if value in d.get(key, '')]
+    
+    def filter_log_Count(self, logs, key, value):
+        return sum(1 for log in logs if value in log[key])
 
     def check_resource_count(self, expected_resource_count):
         resource_count = len(
             list(self.get_resources(self.resource_group_name)))
         self.assertTrue(resource_count == expected_resource_count,
-                        f"resource count of resource group {self.resource_group_name} differs from expected count : {resource_count}")
\ No newline at end of file
+                        f"resource count of resource group {self.resource_group_name} differs from expected count : {resource_count}")
+
+    @classmethod
+    def sumo_query_count(cls, query='_sourceCategory="azure_br_logs" | count', relative_time_in_hours=1):
```

Comment:
make azure_br_logs as variable and use the same variable for source category when we are creating the sumo logic http source.

Reply:
done